src/contacts/EditContact/EditContactContainer.js

```diff
@@ -16,67 +16,73 @@ import { saveContact } from './util';
 import EditContact from './EditContact';
 import { getBackPath } from '../../common/utils/createItem';
 import { DICT_CATEGORIES } from '../../common/constants';
+import { useTranslatedCategories } from '../../common/hooks';
 
-class EditContactContainer extends Component {
-  static manifest = Object.freeze({
-    contact: contactResource,
-    [DICT_CATEGORIES]: categoriesResource,
-    contactsOrg: organizationResource,
-  });
-
-  onClose = (contactId = this.props.match.params.id) => {
-    const { orgId, history, onClose } = this.props;
-
+const EditContactContainer = ({
+  history,
+  match,
+  mutator,
+  onClose,
+  orgId,
+  resources,
+  showMessage,
+}) => {
+  const onCloseForm = useCallback((contactId = match.params.id) => {
     if (onClose) {
       onClose(orgId, contactId);
     } else {
       history.push(getBackPath(orgId, contactId, 'contacts'));
     }
-  };
-
-  onSubmit = (contact) => {
-    const { mutator, resources, showMessage } = this.props;
+  }, [match.params.id, onClose, history, orgId]);
 
-    saveContact(mutator, contact, get(resources, 'contactsOrg.records.0'))
+  const onSubmit = useCallback(contactValues => {
+    saveContact(mutator, contactValues, get(resources, 'contactsOrg.records.0'))
       .then(({ id }) => {
         showMessage('ui-organizations.contacts.message.saved.success', 'success');
-        this.onClose(id);
+        onCloseForm(id);
       })
       .catch(() => showMessage('ui-organizations.contacts.message.saved.fail', 'error'));
-  }
-
-  render() {
-    const { match, resources } = this.props;
-    const isNew = !match.params.id;
-    const loadedContact = get(resources, 'contact.records[0]');
-    const categories = get(resources, `${DICT_CATEGORIES}.records`);
+  },
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  [resources, onCloseForm, showMessage]);
```

Comment:
`resources` is always a new object, `useCallback` makes no sense with it in deps